fe/fe-core/src/main/java/com/starrocks/connector/ScanTableVersion.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.Optional;
+
+public class ScanTableVersion {
```

Comment:
How about naming it as `TableVersionRange`

Reply:
sounds good, this name is consistent with our scanRange.